browser/js/common/directives/add-review/add-review.html

```diff
@@ -0,0 +1,25 @@
+<div class='add-review' ng-show='isLoggedIn()'>
+  <form name="reviewForm">
+
+          <select ng-model="selectedStars"
+          ng-options="star for star in stars"
+          ng-required="true">
+          </select>
+
+        <p ng-show="reviewForm.$invalid && reviewForm.reviewText.$dirty">
```

Comment:
Showing an error message immediately when someone is typing for the first time is abrupt for the user. Try setting this so that it only shows up for the first time once it's `touched`, rather than `dirty`. Additionally, error messages typically look better under the input, not above.